src/main/java/com/epam/ivankaliada/BasicSpringApp/BasicSpringAppApplication.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.ivankaliada.BasicSpringApp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+
+@ComponentScan(basePackages = {"com.epam.ivankaliada.BasicSpringApp"})
+@SpringBootApplication
+@EntityScan("com.epam.ivankaliada.BasicSpringApp.entity")
```

Comment:
Also optional annotation. I believe Spring scans and creates these entities by default.